pkgs/data/misc/iana-etc/default.nix

```diff
@@ -1,17 +1,81 @@
-{stdenv, fetchurl}:
+{ stdenv, fetchurl }:
 
-stdenv.mkDerivation rec {
-  name = "iana-etc-2.30";
+let
 
-  src = fetchurl {
-    url = "http://sethwklein.net/${name}.tar.bz2";
-    sha256 = "03gjlg5zlwsdk6qyw3v85l129rna5bpm4m7pzrp864h0n97qg9mr";
+  iana = {
+    
+    protocolsXML = fetchurl {
+      urls = [
+        "https://www.iana.org/assignments/protocol-numbers/protocol-numbers.xml"
+        "ftp://ftp.iana.org/assignments/protocol-numbers/protocol-numbers.xml"
+      ];
```

Comment:
This url does not looks stable to me. Maybe we can mirror the content somewhere else?